pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -48,7 +50,9 @@ mkDerivation rec {
     qtwebengine
     qtquickcontrols2
     qtgraphicaleffects
+    qtwebsockets
     sqlite
+    libsecret
```

Comment:
Please move `libsecret` up between `libcloudproviders` and `openssl` to maintain the alphabetical order.